ocaml/xapi/repository_helpers.ml

```diff
@@ -817,16 +817,20 @@ let assert_url_is_valid ~url =
     error "Invalid url %s: %s" url (ExnHelper.string_of_exn e) ;
     raise Api_errors.(Server_error (invalid_base_url, [url]))
 
-let gpgkey_path_pattern = Re.Str.regexp "^[A-Za-z0-9_-]+$"
+let is_gpgkey_path_valid = function
```

Comment:
The name is misleading - this functions does not check the validity of a path but is a predicate over characters. The predicate does not include `/` and not `.` - both are typically found in filenames and paths - is this correct?

Reply:
It's a file name under a specific path `/etc/pki/rpm-gpg/`.

Reply:
Right, and I see that we asked you to change "name" into "path" :(

Reply:
And also the predicate was a suggestion in the comments above.